content/copilot/index.md

```diff
@@ -76,6 +93,9 @@ see [Using the extension](#using-the-extension).
 {{< /tab >}}
 {{< /tabs >}}
 
+You can verify that the extension has been properly installed by typing
+`@docker` in the Copilot Chat window.
```

Comment:
We may want to be a bit more explicit here since one customer already struggled with this. Perhaps mentioning that as you type you should see `@docker` appear in the menu?

Reply:
done, added the screengrab too